submissions/DmitryHniezdilov/task-a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,63 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/DmitryHniezdilov/task-a-tiny-JS-world/tree/gh-pages
+   Web app: https://dmitryhniezdilov.github.io/task-a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+
+const dog = {
+  species: "dog",
+  name: "Togo",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "woof-woof!",
+};
+
+const cat = {
+  species: "cat",
+  name: "Murka",
+  gender: "female",
+  legs: 4,
+  hands: 0,
+  saying: "meow-meow!",
+};
+
+const bear = {
+  species: "character",
+  name: "Winnie",
+  gender: "male",
+  legs: 4,
+  hands: 0,
+  saying: "Think, think, think!",
+};
+
+const men = {
```

Comment:
`men` and `women` are plurals. Your objects describe a single man and a single woman.

Reply:
THX for the correction. I'm confused 😕